requirements.txt

```diff
@@ -1,3 +1,4 @@
 mavsdk>=1.4.0
 numpy
+pymavlink
```

Comment:
Lets put a version, if you want also add versions to our other dependencys missing one